app/client/templates/admin/submissionReview.js

```diff
@@ -1,3 +1,25 @@
+Template.submissionReview.helpers({
+
+  breaklines: function (text) {
```

Comment:
this function names don't tell what they do. something like `formatWithBreaklines` is more descriptive. I suck at naming things tho, i think u can come with a better name.

same for `userRating`, this one just need a verb like `getUserRating`.